pipeline/relation_embedder/relation_embedder/scripts/run_local.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <PIPELINE_DATE>"
+  exit 1
+fi
+
+export PIPELINE_DATE=$1
+
+PROJECT_NAME="relation_embedder"
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$DIR"/..
+
+# Read .template.env, substitute variables, and write to .env
+envsubst < template.env > .env
+
+# Build the project
+pushd ../../..
+sbt "project $PROJECT_NAME" ";stage"
+popd
+
+# Build the docker image
+docker compose -f local.docker-compose.yml \
+  build lambda
+
```

Comment:
I'd be tempted to call this a build script and end it here (or have a build and a run script).

Although recompiling with SBT is reasonably efficient, it still takes about 15 seconds to reach this point when there's nothing to do.

(not a blocker)

Reply:
I've added a `--skip-build` flag to do this optionally.